django_ramos/exceptions.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+
+class InvalidBackendError(Exception):
```

Comment:
What do you think about adding some docstrings to the public interface (`InvalidBackendError`, `ThreadSafeCreateMixin`, `BackendPool`, etc.)